src/pages/index.page.jsx

```diff
@@ -17,13 +17,16 @@ export default function Home() {
                 we&#39;re adding new components frequently!
               </p>
             </div>
-            <div>
+            <picture className="home-top__top-image">
+              <source
+                srcset="/images/home-header-image_dark.svg"
+                media="(prefers-color-scheme: dark)"
+              />
```

Comment:
If I remove that `className` from `picture`, and put that `className` on the image instead, it behaves just like light-mode version. 🤷‍♀️

I know you said there had to be the class on `picture`, so I'm not sure why that works. This might just be a quick fix before we do a card for the images like we talked about during our rebasing.